core/routemgmt/common/apigw-utils.js

```diff
@@ -566,7 +566,7 @@ function setActionOperationInvocationDetails(swagger, endpoint, operationId, res
   var caseIdx = getCaseOperationIdx(caseArr, operationId);
   var operations = [operationId];
   _.set(swagger, 'x-ibm-configuration.assembly.execute[0].operation-switch.case['+caseIdx+'].operations', operations);
-  _.set(swagger, 'x-ibm-configuration.assembly.execute[0].operation-switch.case['+caseIdx+'].execute[0].invoke.target-url',  makeWebActionBackendUrl(endpoint.action, responsetype, getPathParameters(endpoint.gatewayPath)));
+  _.set(swagger, 'x-ibm-configuration.assembly.execute[0].operation-switch.case['+caseIdx+'].execute[0].invoke.target-url',  makeWebActionBackendUrl(endpoint.action, responsetype) + (responsetype === 'http' ? "$(request.path)" : ""));
```

Comment:
ok.. i understand why this was added here and not in the `makeWebActionBackendUrl` function..  do you recall the failure when it was added in the function?

Reply:
Not a failure, but the path parameters where not need it.
The failure was a test because it was adding the `$(request.path)` in both sections of swagger doc 
1. x-openwhisk
2. execute/target-url
I think we only want in the 2nd (execute/target-url) because 1st section (x-openwhisk) is decoration section for API Gateway UI integration, and we don't want to show the extra `$(request.path)` there.